lighthouse-core/config/default-config.js

```diff
@@ -472,6 +473,7 @@ const defaultConfig = {
         {id: 'long-tasks', weight: 0, group: 'diagnostics'},
         {id: 'non-composited-animations', weight: 0, group: 'diagnostics'},
         {id: 'unsized-images', weight: 0, group: 'diagnostics'},
+        {id: 'jankless-font', weight: 0, group: 'diagnostics'},
```

Comment:
so based on the confirmation in #11227 the advice in this audit is actually to improve the chances that the font gets used and doesn't really have any impact on the performance metrics or CLS (the layout shift doesn't happen in latest chrome whether you preload or not)

given that, I'm thinking it might better belong in "Best Practices - UX" group

Reply:
Good point! I'll move it to the best practices UX group! However, should I set it to be weight 0 or 1? I am leaning towards 1 since I feel that this is niche enough it won't make a big difference to most users (have yet to confirm that though)

Reply:
I agree weight 1 makes sense, if we end up finding edge cases where this isn't a good idea then we can always reevaluate :)

Reply:
wait–do we not consider score changes for non-perf categories to be breaking changes?

Reply:
Nope, I consider them breaking for LHCI which you might be remembering? But we've already added a few audits since 6.0 to best practices IIRC.